source/reference/limitations.txt

```diff
@@ -64,6 +64,36 @@ General Limitations
 - You cannot sync a collection that has a unique index and a non-unique
   index defined on the same field(s).
 
+.. _c2c-limitations_multiple_sync:
+
+Multiple Source Sync Limitations
+--------------------------------
+
+.. include:: /includes/many-with-one-cluster.rst
+
+A namespace is a ``database_name.collection_name`` combination. You can
+only sync namespaces that don't conflict.
+
+For example, consider this scenario:
+
+- Two source clusters S1 and S2.
+- A destination cluster D.
+- Databases ``X`` and ``Y`` on both S1 and S2.
+- Collections ``A``, ``B``, ``C``, and ``D`` on both S1 and S2.
+- You can sync both of these combinations:
+
+  - ``X.A`` and ``Y.C`` on S1 with D.
+  - ``X.B`` and ``Y.D`` on S2 with D.
+
```

Comment:
**Suggestion**

I would move the first sentence of the explanation here:

```
If D is initially empty, you can sync S1 with D. ``X.A`` and ``X.B`` are copied from S1 to D.
```

It took me a while to understand the concept here, the reason you cannot migrate the 2nd set of name spaces is because the namespaces were already taken as part of this sync.

Reply:
Done.